bugshaker/src/main/java/com/github/stkent/bugshaker/flow/email/EmailCapabilitiesProvider.java

```diff
@@ -54,20 +54,6 @@ public EmailCapabilitiesProvider(
         this.logger = logger;
     }
 
-    public boolean canSendEmails() {
-        logger.d("Checking for email apps...");
```

Comment:
Ahh just re-read this class, lets look at if we are checking for email sending capabilities.